### PR TITLE
Add New Expressions

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/refguide/modeling/application-logic/expressions/add-date-function-calls.md [hunk 1]
```diff
@@ -259,11 +259,58 @@ The output is:
 "Mon Feb 01 01:01:01 CET 2008"
 ```
 
-## 9 addYears[UTC]
+## 9 addQuarters
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/application-logic/expressions/add-date-function-calls.md]: @vdyachenko, I added non-UTC one but we mention only UTC when there is an alternative. Shall I delete it?

File: content/en/docs/refguide/modeling/application-logic/expressions/subtract-date-function-calls.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+subtractHours(dateTime(2007, 1, 1, 1, 1, 1), 1)
+```
+
+The output is:
+
+```java
+Mon Jan 01 00:01:01 CET 2007
+```
+
+## 6 subtractDays and subtractDays[UTC]
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/application-logic/expressions/subtract-date-function-calls.md]: @vdyachenko, shall I leave only UTC ones just like in the Add Date Function Calls document?

**Comment by MariaShaposhnikova** [comment]
@vdyachenko, please review this PR.

File: content/en/docs/refguide/modeling/application-logic/expressions/subtract-date-function-calls.md (hunk 2 above)
Comment (vdyachenko) [reply content/en/docs/refguide/modeling/application-logic/expressions/subtract-date-function-calls.md]: It should look the same as we have in `Add Date Function Calls`. There we have `addDays[UTC]` title, so here we should have `subtractDays[UTC]` title.

File: content/en/docs/refguide/modeling/application-logic/expressions/add-date-function-calls.md (hunk 1 above)
Comment (vdyachenko) [reply content/en/docs/refguide/modeling/application-logic/expressions/add-date-function-calls.md]: Same here, it should be `addQuarters[UTC]`, same as `addMonths[UTC]`, `addDays[UTC`, etc.

**Pull request merged**